src/legacy/core_plugins/kibana/public/selectors/dashboard_selectors.ts

```diff
@@ -46,7 +46,9 @@ export const getEmbeddableStagedFilter = (state: CoreKibanaState, panelId: Panel
 export const getEmbeddableMetadata = (state: CoreKibanaState, panelId: PanelId) =>
   DashboardSelectors.getEmbeddableMetadata(getDashboard(state), panelId);
 
-export const getStagedFilters = (state: CoreKibanaState): Filters =>
+export const getStagedFilters = (
```

Comment:
staged filters are a different shape than regular filters.  This code will go away with new Embeddable stuff, so not creating a type for it here.